packages/analytics/client/src/analytics_client/analytics_client.ts

```diff
@@ -171,6 +171,7 @@ export class AnalyticsClient implements IAnalyticsClient {
     this.optInConfig$.next(optInConfigInstance);
   };
 
+  // @ts-expect-error
```

Comment:
Why do we need that now? Can we at least have a description or a reason? I see a lot of those in our owned changes.